src/cs/ui/UI.cs

```diff
@@ -502,6 +518,16 @@ public enum InfoType { W_ENGERGY, S_ENGERGY, SUPPORT, ENVIRONMENT, POLLUTION, MO
 		MoneyL.Text = Data.Money.ToString();
 		ImportCostL.Text = Data.Imports.ToString();
 	}
+	
+	// Sets the correct years on the Next Turn Animation
+	public void SetNextYears() {
+		Year = Timeline.Value;
+		var Anim = TimelineAP.GetAnimation("NextTurnAnim");
+		var Track = Anim.FindTrack("Year:text", Animation.TrackType.Value);
+		var NKeys = Anim.TrackGetKeyCount(Track);
```

Comment:
Please avoid type inference (use of `var`) wherever possible, it makes the code really hard to debug.